vault/core.go

```diff
@@ -1573,10 +1575,8 @@ func (c *Core) postUnseal() (retErr error) {
 	c.clearForwardingClients()
 	c.requestForwardingConnectionLock.Unlock()
 
-	// Purge the backend if supported
-	if purgable, ok := c.physical.(physical.Purgable); ok {
-		purgable.Purge(c.activeContext)
-	}
+	c.physicalCache.Purge(c.activeContext)
+	c.physicalCache.SetEnabled(true)
```

Comment:
I think we want to conditionally enable it based on if we have caching disabled in the config

Reply:
Oops yes you're right. I thought about that before and then forgot to do it.

Reply:
Fixed!